pyppeteer/target.py

```diff
@@ -84,7 +107,8 @@ def type(self) -> str:
         ``'browser'``, or ``'other'``.
```

Comment:
We can type this (can't comment on non-changed line, unfortunately): 
`def type(self) -> Literal['page','background_page','service_worker','browser','other']:`
Since that's a bit unweildly, you could even declare the type outside of the class somewhere to keep it clean.

Reply:
This was also only added in 3.8 so we'd need mypy_extensions.

Reply:
Great idea. We should probably work on reviewing and improving all of type hints after 2.1.1